kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -21,8 +21,8 @@
           <label for="username">Password</label>
           <input type="password" id="password" required v-model="user.password">
         </div>
-
         <div class="user-field">
+          <label for="role">Role</label>
           <select v-model="user.role">
```

Comment:
good job adding the label - but you need to give the `select` dropdown `name="role"` and `id="role"` for the label to be associated with it